tomcat-main/src/main/resources/defaults/base-procedure-hit.xml

```diff
@@ -0,0 +1,86 @@
+  </section>
+
+  <section id="hitInformation">
+    <field id="hitNumber" mini="number,list" />
+    <field id="entryDate" datatype="date" />
+
+    <repeat id="hitDepositorGroupList/hitDepositorGroup">
+      <field id="depositor" mini="summary,list" autocomplete="true" />
+      <field id="depositorContact" autocomplete="true" />
+      <field id="depositorContactType" autocomplete="true" ui-type="enum" />
+      <field id="depositorNote" />
+    </repeat>
+
+    <repeat id="agreementGroupList/agreementGroup">
+      <field id="agreementStatus" ui-search="repeatable" autocomplete="true" ui-type="enum" />
+      <field id="agreementStatusDate" ui-search="repeatable" datatype="date" />
+      <field id="agreementStatusNote" ui-search="repeatable" />
+    </repeat>
+
+    <repeat id="entryMethods">
+      <field id="entryMethod" autocomplete="true" ui-type="enum" />
+    </repeat>
+
+    <repeat id="agreementRenewalDates">
+      <field id="agreementRenewalDate" datatype="date" ui-search="repeatable" />
+    </repeat>
+
+    <field id="entryReason" autocomplete="true" ui-type="enum" />
+    <field id="returnDate" datatype="date" />
+    <field id="entryNote" />
+
+    <repeat id="internalApprovalGroupList/internalApprovalGroup">
+      <field id="internalApprovalGroupName" autocomplete="true" />
+      <field id="internalApprovalIndividual" autocomplete="true" />
+      <field id="internalApprovalStatus" autocomplete="true" ui-type="enum" />
+      <field id="internalApprovalDate" datatype="date" />
+      <field id="internalApprovalNote" />
+    </repeat>
+
+    <repeat id="externalApprovalGroupList/externalApprovalGroup">
+      <field id="externalApprovalGroupName" autocomplete="true" />
+      <field id="externalApprovalIndividual" autocomplete="true" />
+      <field id="externalApprovalStatus" autocomplete="true" ui-type="enum" />
+      <field id="externalApprovalDate" datatype="date" />
+      <field id="externalApprovalNote" />
+    </repeat>
+  </section>
+
+  <section id="cultureCareAndHandling">
+    <field id="handlingPreferences" />
+    <repeat id="handlingLimitationsGroupList/handlingLimitationsGroup">
+      <field id="handlingLimitationsType" autocomplete="true" ui-type="enum" />
+      <field id="handlingLimitationsLevel" autocomplete="true" ui-type="enum" />
+      <field id="handlingLimitationsDetail" />
+      <field id="handlingLimitationsRequestor" autocomplete="true" />
+      <field id="handlingLimitationsOnBehalfOf" autocomplete="true" />
+      <field id="handlingLimitationsDate" datatype="date" />
+    </repeat>
+  </section>
+
+  <section id="correspondence">
+    <repeat id="correspondenceGroupList/correspondenceGroup">
+      <field id="correspondenceDate" datatype="date" />
+      <field id="correspondenceSender" autocomplete="true" />
+      <field id="correspondenceRecipient" autocomplete="true" />
+      <field id="correspondenceSummary" />
+      <field id="correspondenceReference" ui-type="enum" />
```

Comment:
This is free text in the UI, so remove `ui-type="enum"` (which I don't think would do anything anyway, without `autocomplete="true"` also being present.).